owlkettle/widgets.nim

```diff
@@ -3525,6 +3525,98 @@ renderable Scale of BaseWidget:
         echo "New value is ", newValue
         app.value = newValue
 
+
+# See TODO at comment of PixbufObj regarding why we wrap GtkMediaStream with MediaStreamObj
+type 
+  MediaStreamObj = object
+    gdk: GtkMediaStream
```

Comment:
In that case quick question regarding pattern:
In the other objects we called this field gdk. 
Is the fieldname supposed to resemble the name of the library where the object is from?

Reply:
The idea is basically: `MediaStream` is the "Owlkettle object" and `MediaStream.gtk` is the "GTK object" belonging to that owlkettle object. It could also be called something like `native`, `internal` (as with internalWidget) or `extern`, but the naming convention for the other wrapper objects was using `gdk`/`gtk` depending on the types origin.